File: java/src/main/java/gov/doc/ntia/sigmf/ext/global/location/GeographicCoordinateSystem.java
package gov.doc.ntia.sigmf.ext.global.location;

import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonProperty;

@JsonInclude(JsonInclude.Include.NON_NULL)
public class GeographicCoordinateSystem extends CoordinateSystem {



    @JsonProperty(value = "datum", required = false)
    protected Datum datum;


    public Datum getDatum() {
        return datum;
    }

    public void setDatum(Datum datum) {
        this.datum = datum;
    }







}

File: Csharp/sigmf/sigmf/ext/global/location/GeographicCoordinateSystem.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Text;

namespace sigmf
{
    public class GeographicCoordinateSystem : CoordinateSystem
    {
        [JsonProperty("datum")]
        protected Datum datum;

        public Datum Datum { get => datum; set => datum = value; }
    }
}
